config/configgrpc/configgrpc.go

```diff
@@ -172,6 +172,16 @@ func (gcs *ClientConfig) SanitizedEndpoint() string {
 	}
 }
 
+func (gcs *ClientConfig) EndpointPort() (string, error) {
```

Comment:
We're working towards GA for `configgrpc` right now and I am hesitant to add new public APIs. For now, let's move this function into the otlpexporter.

Reply:
Addressed in ec2b9d3